scrips/calendarVars.js

```diff
@@ -2,5 +2,18 @@ const weekDaysNumber = 7;
 const hoursNumber = 24;
 const cellHeight = 49;
 const currentEventTileId = "current-event-tile";
+const defaultEventLength = 0.75; //hours
+const emptyEventTitle = "(no title)";
+const minutesGap = 15;
+const minutesIncrement = [];
+for (let i = 0; i < 60 / minutesGap; i++) minutesIncrement.push(i * minutesGap);
```

Comment:
Constants are values which are not static. Here you are doing computing things. It should be utility function
```suggestion
const getMinutesIncrement = () => {
  const minutesIncrement = [];
  
  for (let i = 0; i < 60 / minutesGap; i++) {
    minutesIncrement.push(i * minutesGap)
  };
  
  return minutesIncrement;
};
```

P.S I can not tell what is a use case for this variable from the name